src/main/java/com/ibm/cloud/cloudant/kafka/schema/ConnectRecordMapper.java

```diff
@@ -33,6 +33,9 @@
     private static Logger LOG = LoggerFactory.getLogger(ConnectRecordMapper.class);
 
     public Map<String, Object> apply(ConnectRecord<R> record) {
+        if (record.value() == null) {
```

Comment:
this came up in testing - maybe left over from when i was looking at tombstones? anyway, it's probably useful to have